init-letsencrypt.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+if ! [ -x "$(command -v docker-compose)" ]; then
+  echo 'Error: docker-compose is not installed.' >&2
+  exit 1
+fi
+
+domains=(diamonds.etimo.se www.diamonds.etimo.se)
+rsa_key_size=4096
+data_path="./data/certbot"
+email="philip.forsberg@etimo.se"
```

Comment:
Dont add your personal email to our public repo. And we should probably not use it for production as well, perhaps we can use kontakt@etimo.se or something similar?

Reply:
Fixed!